docs/topics/crud-mission-database-interaction.adoc

```diff
@@ -1,5 +1,5 @@
 [[common-crud-database-interaction]]
-= Interacting with the CRUD API
+= Interacting with the Relational Database Backend API
```

Comment:
No. The API really only exposes CRUD functionality. It doesn't directly expose the relational database (obviously). Maybe something like "Interacting with the application API"?